AlembicImporter/Assets/InitTestScene636656905069999691.unity.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
please remove this file and the accompanying Unity scene. I believe these are auto-generated right?